proto/viam/app/data/v1/data.proto

```diff
@@ -244,6 +247,19 @@ message DeleteTabularDataByFilterResponse {
   reserved "result";
 }
 
+// DeleteTabularDataRequest deletes the data from the organization that is older than `delete_before_days`.
+// If `delete_before_days` is 10, this deletes any data that was uploaded up to 10 days ago. If it is 0 or
```

Comment:
```suggestion
// For example if `delete_before_days` is 10, this deletes any data that was uploaded up to 10 days ago. If it is 0 or
```
Super nit it just made it sound like there was something special about the number 10 rather than this being an example